lib/instance.js

```diff
@@ -634,10 +634,15 @@ Instance.prototype.save = function(options) {
         var instance = self.get(include.as);
         if (!instance) return Promise.resolve();
 
-        return instance.save({
+        var includeOptions = _.defaults(include.options || {}, {
           transaction: options.transaction,
           logging: options.logging
-        }).then(function () {
+        });
+        _.assign(includeOptions, {
+          parentRecord: self
+        });
```

Comment:
Yep, the `defaults` is because `transaction` and `logging` can be override, and `assign` because `parentInstance` not. Would you like to force the parent `transaction` and `logging` options?

What about inheriting options from the overall create? Which ones?